core/tabs/applications-setup/browsers/waterfox.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh -e
+
+. ../../common-script.sh
+
+installWaterfox() {
+    if ! command_exists waterfox; then
+        printf "%b\n" "${YELLOW}Installing waterfox...${RC}"
+        case "$PACKAGER" in
+            pacman)
+		"$AUR_HELPER" -S --needed --noconfirm waterfox-bin
+                ;;
+            *)
+		. ./setup-flatpak.sh
```

Comment:
@nnyyxxxx You're the one not testing now. 

```suggestion
		. ../setup-flatpak.sh
```

Reply:
> @nnyyxxxx You're the one not testing now.

fixed

Reply:
> @nnyyxxxx You're the one not testing now.

not my job, not my pr, :laughing: